src/solana/WalletSignBottomSheetTransaction.tsx

```diff
@@ -205,7 +205,11 @@ const WalletSignBottomSheetTransaction = ({
             header="Simulation Failed"
             body={`${JSON.stringify(transaction.error)}\n${
               transaction.logs
-                ? transaction.logs[transaction.logs.length - 1]
+                ? transaction.logs.some((log) =>
+                    log.includes('SlippageToleranceExceeded'),
+                  )
+                  ? 'Slippage tolerance exceeded. Please increase slippage'
```

Comment:
can this message be put into strings file? Hate the jargon but not sure what a better message is right now.

Reply:
Ah yeah, probably should go in the translations file

Reply:
Fixed